homeassistant/components/webdav/camera.py

```diff
@@ -0,0 +1,235 @@
+"""
+This component models a WebDAV share full of images as a camera.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.webdav/
+"""
+
+import logging
+
```

Comment:
Please remove this blank line. I recommend the isort tool to automatically sort and group imports.